genie-web/src/main/java/com/netflix/genie/web/tasks/job/JobMonitor.java

```diff
@@ -205,14 +218,22 @@ public void run() {
      */
     @Override
     public GenieTaskScheduleType getScheduleType() {
-        return GenieTaskScheduleType.FIXED_DELAY;
+        return GenieTaskScheduleType.ADAPTIVE_BACKOFF;
```

Comment:
Should this be parameterized with something like fast property. In case there is an issue we can quickly fall back to FIXED_DELAY

Reply:
Currently we don't support fast properties so it would require a reboot anyway. We could make it a property what the "default" is